pythonforandroid/recipes/host_sh/__init__.py

```diff
@@ -0,0 +1,16 @@
+from pythonforandroid.recipe import PythonRecipe
+
+
+class Host_sh_recipe(PythonRecipe):
```

Comment:
Could you change these recipes to follow pep8, e.g. this one would be HostShRecipe